src/components/SignupCard/index.jsx

```diff
@@ -1,31 +1,33 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { Link, useHistory } from 'react-router-dom';
 
-import { Button, Card, Form, Input, Space, Typography } from 'antd';
+import { Button, Card, Form, Input, Modal, Space, Typography } from 'antd';
 import { LockOutlined, UserOutlined, MailOutlined } from '@ant-design/icons';
 
-import './SignupCard.scss';
-import { useAuth } from 'services/AuthProvider';
+import { useAuth } from 'hooks/Hooks';
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports